pkg/gardener/shoot/extender/provider/provider_test.go

```diff
@@ -103,7 +105,7 @@ func TestFixAlignWorkerZonesWithGardener(t *testing.T) {
 					Provider: fixProviderWithMultipleWorkersAndConfig(hyperscaler.TypeAWS, fixMultipleWorkers([]workerConfig{
 						{"main-worker", "m6i.large", "gardenlinux", "1310.4.0", 1, 3, []string{"eu-central-1a", "eu-central-1b", "eu-central-1c"}},
 						{"additional", "m7i.large", "gardenlinux", "1311.2.0", 2, 4, []string{"eu-central-1a"}},
-					}), fixAWSInfrastructureConfig("10.250.0.0/22", []string{"eu-central-1a", "eu-central-1b", "eu-central-1c"}), fixAWSControlPlaneConfig()),
+					}), fixAWSInfrastructureConfig(t, "10.250.0.0/22", []string{"eu-central-1a", "eu-central-1b", "eu-central-1c"}), fixAWSControlPlaneConfig()),
 					Networking: imv1.Networking{
```

Comment:
The test "Create provider config for multiple workers with AWS worker config provided externally" should be removed as we are making deprecated fields`spec.shoot.provider.infrastructureConfig` and `spec.shoot.provider.controlPlaneConfig`

Reply:
Done